docs/modules/ROOT/nav.adoc

```diff
@@ -27,45 +28,55 @@
-** xref:user-guide/ssl.adoc[SSL, TLS and HTTPS]
-** xref:user-guide/auth.adoc[Authentication and authorization]
-** xref:security-guide/ws-securitypolicy-auth.adoc[WS-SecurityPolicy Authentication]
+** xref:user-guide/ssl-tls-https.adoc[SSL, TLS and HTTPS]
+** xref:user-guide/authentication-authorization.adoc[Authentication and authorization]
+** xref:security-guide/ws-securitypolicy-authentication-authorization.adoc[WS-SecurityPolicy Authentication]
+
+// release-notes
 * xref:release-notes/index.adoc[Release notes]
+
 ifeval::[{doc-is-main} == true]
-** xref:release-notes/3.15.0.adoc[3.15.0 LTS]
-** xref:release-notes/3.14.0.adoc[3.14.0]
-** xref:release-notes/3.13.1.adoc[3.13.1]
-** xref:release-notes/3.13.0.adoc[3.13.0]
-** xref:release-notes/3.12.0.adoc[3.12.0]
-** xref:release-notes/3.11.1.adoc[3.11.1]
-** xref:release-notes/3.11.0.adoc[3.11.0]
-** xref:release-notes/3.10.0.adoc[3.10.0]
-** xref:release-notes/3.9.0.adoc[3.9.0]
-** xref:release-notes/3.8.6.adoc[3.8.6 LTS]
-** xref:release-notes/3.8.5.adoc[3.8.5 LTS]
-** xref:release-notes/3.8.4.adoc[3.8.4 LTS]
-** xref:release-notes/3.8.3.adoc[3.8.3 LTS]
-** xref:release-notes/3.8.2.adoc[3.8.2 LTS]
-** xref:release-notes/3.8.1.adoc[3.8.1 LTS]
-** xref:release-notes/3.8.0.adoc[3.8.0 LTS]
-** xref:release-notes/2.7.1.adoc[2.7.1]
-** xref:release-notes/2.7.0.adoc[2.7.0]
-** xref:release-notes/2.6.1.adoc[2.6.1]
-** xref:release-notes/2.6.0.adoc[2.6.0]
-** xref:release-notes/2.5.0.adoc[2.5.0]
-** xref:release-notes/2.4.0.adoc[2.4.0]
-** xref:release-notes/2.3.1.adoc[2.3.1]
-** xref:release-notes/2.3.0.adoc[2.3.0]
-** xref:release-notes/2.2.2.adoc[2.2.2 LTS]
-** xref:release-notes/2.2.1.adoc[2.2.1 LTS]
-** xref:release-notes/2.2.0.adoc[2.2.0 LTS]
+** xref:release-notes/rn-3-15-0.adoc[3.15.0 LTS]
+** xref:release-notes/rn-3-14-0.adoc[3.14.0]
+** xref:release-notes/rn-3-13-1.adoc[3.13.1]
+** xref:release-notes/rn-3-13-0.adoc[3.13.0]
+** xref:release-notes/rn-3-12-0.adoc[3.12.0]
+** xref:release-notes/rn-3-11-1.adoc[3.11.1]
+** xref:release-notes/rn-3-11-0.adoc[3.11.0]
+** xref:release-notes/rn-3-10-0.adoc[3.10.0]
+** xref:release-notes/rn-3-9-0.adoc[3.9.0]
+** xref:release-notes/rn-3-8-6.adoc[3.8.6 LTS]
+** xref:release-notes/rn-3-8-5.adoc[3.8.5 LTS]
+** xref:release-notes/rn-3-8-4.adoc[3.8.4 LTS]
+** xref:release-notes/rn-3-8-3.adoc[3.8.3 LTS]
+** xref:release-notes/rn-3-8-2.adoc[3.8.2 LTS]
+** xref:release-notes/rn-3-8-1.adoc[3.8.1 LTS]
+** xref:release-notes/rn-3-8-0.adoc[3.8.0 LTS]
+** xref:release-notes/rn-2-7-1.adoc[2.7.1]
+** xref:release-notes/rn-2-7-0.adoc[2.7.0]
+** xref:release-notes/rn-2-6-1.adoc[2.6.1]
+** xref:release-notes/rn-2-6-0.adoc[2.6.0]
+** xref:release-notes/rn-2-5-0.adoc[2.5.0]
+** xref:release-notes/rn-2-4-0.adoc[2.4.0]
+** xref:release-notes/rn-2-3-1.adoc[2.3.1]
+** xref:release-notes/rn-2-3-0.adoc[2.3.0]
+** xref:release-notes/rn-2-2-2.adoc[2.2.2 LTS]
+** xref:release-notes/rn-2-2-1.adoc[2.2.1 LTS]
+** xref:release-notes/rn-2-2-0.adoc[2.2.0 LTS]
```

Comment:
I would not mind the short prefix, but why not the verbatim version with dots in the name? It is much easier to copy/paste/search/replace when looking for/editing the stuff.